webview-ui/src/components/ResolutionsPage/ResponseWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react";
+import { Label, FlexItem } from "@patternfly/react-core";
+
+interface ResponseWrapperProps {
+  type: "sent" | "received";
+  children: React.ReactNode;
+  className?: string;
+}
+
+export const ResponseWrapper: FC<ResponseWrapperProps> = ({ type, children, className = "" }) => {
```

Comment:
Why not just have 2 components:
  - SentMessage
  - ResponseMessage

I have it in the back of my mind to end up with a few kinds of response/activity messages and it would be nice to have multiple render components based on those types.   (ResponseStringMessage, ResponseMarkdownMessage, ResponseFancyPantsFormattedMessage, ...)